### ISSUE TITLE
Npm link cause "Unable to resolve module `AccessibilityInfo` "

### ISSUE THREAD

**Issue by Kroderia** [facebook/react-native#15172]
### Is this a bug report?

Feels like it is a bug.

### Environment

1. `react-native -v`: `react-native-cli: 2.0.1
react-native: 0.46.4
`
2. `node -v`:`v8.1.3
`
3. `npm -v`:`5.3.0
`

- Target Platform: *

- Development Operating System: macOS 10.12.5 & Windows 10 64bit

### Steps to Reproduce

There is a RN Project A and Denpendency B.
B is abstracted from A, it works properly before.
Now B has a dependency on RN.
Project A install Dependency B using `npm link ../B && npm install ../B --save`.

Structure as below:
 |--A  
 |--|--node_modules  
 |-------|--RN  
 |-------|--B  
 |----------|--node_modules  
 |---------------|--RN  

Run with `react-native start`

### Expected Behavior

Everything goes fine.

### Actual Behavior

Error pops up as `error: bundling failed: "Unable to resolve module "AccessibilityInfo" from "/path/to/A/node_modules/react-native/Libraries/react-native/react-native-implementation.js": Module does not exist in the module map`.

**Comment by hramos** [comment]
Duplicate of #14209

**Comment by lucienbl** [comment]
I don't know if someone told that before me but downgrading from 0.56.0 to 0.55.4 solved this issue for me. 😃

**Comment by Chris-90** [comment]
I don't think that it is a dublicate of an multiple years old issue. Because the only solution of the old issue was to downgrade version.
Downgrading is only a temporary fix, but no permanent solution. 
The problem seems to be the same, it doesn't mean that reason is identacally, too.

**Comment by ronyoka** [comment]
It seems the issue has been present in all kind of operating system and react version for years. And the support is very unprofessional...

**Comment by anothercoder2** [comment]
Same issue here. Did something change in the last two days?

**Comment by hramos** [comment]
Please submit a new issue, as this one is closed.
